docs/reference/train.tf_estimator.html

```diff
@@ -171,6 +171,11 @@ <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a> Ar
     </tr>
     </table>
     
+    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>
+
+    <p>A data.frame of the training loss history. Note that this will be
+a empty data.frame if <code><a href='hook_history_saver.html'>hook_history_saver()</a></code> is not used as one of the hooks provided.</p>
```

Comment:
Should we attach the `hook_history_saver()` by default? (seems like just accumulating the metrics shouldn't have any performance problems)

Reply:
We could but how frequent should we save? Right now we have `every_n_step` but in the future it's possible to add things like `every_n_second`. What should the default be?

Reply:
If all the history does is update some vectors it seems like it's not a big deal to save it frequently (the worst that can happen is too many points to plot). Maybe every 10 steps?

Reply:
Is the progress bar hook also automatically applied? (it is for Keras).

Reply:
Got it. I just addressed those in https://github.com/rstudio/tfestimators/pull/111/commits/42ac84633d375f11a022301c9311b27cc2c24721!